spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/DnnGraph.scala

```diff
@@ -392,11 +392,13 @@ class DnnGraph(
    */
   private def fusion(): Unit = {
     if (!this.train) {
-      for (j <- 0 to 1) {
+      for (j <- 0 to 2) {
         var i = forwardExecution.length - 1
```

Comment:
why 3 times

Reply:
Should have 3 passes. the second will set the flag about the input whether is negative or not.